root/static/styles/layout.less

```diff
@@ -241,17 +241,17 @@ pre code {
                 list-style-image: none;
                 &:hover {
                     background-color: @musicbrainz-orange;
-                    > a {
+                    > a, .menu-header {
```

Comment:
You forgot the admin menu, that's still a link. But speaking of, /admin doesn't seem like a very useful page, it's just the menu in page-form...

But yeah, I guess we can remove the `a, ` here after.